code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm

```diff
@@ -213,12 +229,20 @@
 	resize = 1.3
 	update_transform()
 
+/mob/living/simple_animal/hostile/slime/big/CanAttack(atom/the_target)
```

Comment:
Big slime inherits this and all other procs from normal slimes already, you can safely delete it.